### DISCUSSION TITLE
Cross over and cross under for moving averages

### DISCUSSION THREAD

**Discussion by hnmange** [DaveSkender/Stock.Indicators#567] [Q&A] [answered]
Hi All, 

Just wanted to know whether it has in-built functions for checking cross over and cross under of two moving averages.

Thank you.

**Comment by DaveSkender** [comment, marked as answer]
No, this library is only generating the indicator data -- we're sticking to the "nothing more" [single-responsibility principle](https://en.wikipedia.org/wiki/Single-responsibility_principle).  The universe of what people do with indicator data is too large to consider.  The universe of possible trade signals alone is uncountable.  I do have some thoughts on other related projects here: #484

**Reply by hnmange** [reply]
Got it. Thanks.
